privaterelay/settings.py

```diff
@@ -660,62 +663,44 @@ def _get_initial_middleware() -> list[str]:
     "usd": {
         "en": {
             "monthly": {
-                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
-                "price": "$?.??",
+                "id": PHONE_PLAN_ID_US_MONTHLY,
+                "price": "$4.99",
             },
             "yearly": {
-                "id": "price_1LXUhcJNcmPzuWtRHUFHVk12",
-                "price": "$?.??",
+                "id": PHONE_PLAN_ID_US_YEARLY,
+                "price": "$3.99",
             },
         },
-        # TODO: Get plan ID for Canada
-        # "ca": {
-        #     "monthly": {
-        #         "id": "",
-        #         "price": "$?.??",
-        #     },
-        #     "yearly": {
-        #         "id": "",
-        #         "price": "$?.??",
-        #     },
-        # },
     },
 }
 PHONE_PLAN_COUNTRY_LANG_MAPPING = {
     "us": {
         "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
     },
-    # Commented out; no plan ID known yet
-    # "ca": {
-    #     "en": PHONE_PLAN_ID_MATRIX["usd"]["ca"],
-    # },
+    "ca": {
+        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
+    },
 }
 
 BUNDLE_PLAN_ID_MATRIX = {
     "usd": {
         "en": {
             "yearly": {
-                "id": "price_1La3d7JNcmPzuWtRn0cg2EyH",
+                # To allow testing the subscription flow on stage, we can set
+                # a custom plan ID via an environment variable:
+                "id": BUNDLE_PLAN_ID_US,
                 "price": "$?.??",
             },
         },
```

Comment:
Do we have a monthly price? It's in the add on:
<img width="330" alt="image" src="https://user-images.githubusercontent.com/13066134/191298243-0689816d-7467-4490-804b-bb534cc1c6ad.png">

Reply:
@codemist "Monthly" and "yearly" refer to the billing period, but the value for `price` is the price per month. (So the actual price is 12 times that.)

Btw, note that in [this follow-up PR](https://github.com/mozilla/fx-private-relay/pull/2507), the price will be changed to a number, so this would look like:

```js
            "yearly": {
                "id": "price_1La3d7JNcmPzuWtRn0cg2EyH",
                # To allow testing the subscription flow on stage, we can set
                # a custom plan ID via an environment variable:
                "id": BUNDLE_PLAN_ID_US,
                "price": 1337,
                "currency": "USD",
            },
```

This can then be formatted using [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat).

Reply:
Oh okay! Got it, thanks.

Reply:
For a second I was confused about `1337` and then I googled it, lol.

Reply:
Haha sorry, should've realised that maybe that wasn't a thing anymore in 2022 :P